### DISCUSSION TITLE
New datalist field type

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2899] [Feature Requests]
Sometimes I need a `select` field that lets users override the built-in options. Fortunately `datalist` is sufficiently supported by browsers, and I've been using it in my channel forms, but I'd love to see it as a core field type.

**Comment by eric-lamb62** [comment]
@murtaugh I put a thing together that should handle this. Can you kick the tires when able?
https://github.com/mithra62/ee-datalist-ft

May not need to be a core function :)

**Comment by murtaugh** [comment]
Right on! I will try it out tomorrow.

**Comment by murtaugh** [comment]
@mithra62 I'm getting the following immediately after dropping the files in place:

`Failed opening required 'user/addons/datalist/vendor/autoload.php' (include_path='.:')`

**Comment by eric-lamb62** [comment]
@murtaugh did you download the files directly or get from Releases? 
https://github.com/mithra62/ee-datalist-ft/releases/tag/1.0.0 <- should work

From the repo, it doesn't have any third party files.

**Comment by murtaugh** [comment]
@mithra62 Hahaha of course I did haha it works fine, thanks
